pincer/client.py

```diff
@@ -852,10 +860,27 @@ async def get_channel(self, _id: int) -> Channel:
         """
         return await Channel.from_id(self, _id)
 
+    async def get_message(self, _id: Snowflake, channel_id: Snowflake) -> UserMessage:
```

Comment:
Shouldn't we use `Union[int, Snowflake]`?

Reply:
Snowflake inherits from int and since it's a subclass I assume it shouldn't matter.

Reply:
`int` isn't "supposed" to be passed in so I agree with @Endercheif